### PR TITLE
Merge scala/stream/stream-customize.md and java/stream/stream-customi…

### PR DESCRIPTION
**Pull request by atiqsayyed**
Refs #23052

Identical contents in scala/stream/stream-customize.md and java/stream/stream-customize.md merged

### CODE REVIEW

**Comment by ktoso** [comment]
OK TO TEST

**Comment by ktoso** [comment]
Whoops, misclicked sorry :) Thanks a lot!

File: akka-docs/src/main/paradox/java/stream/stream-customize.md [hunk 1]
```diff
@@ -1,414 +1 @@
...
-
-If a stage manages a resource with a lifecycle, for example objects that need to be shutdown when they are not
-used anymore it is important to make sure this will happen in all circumstances when the stage shuts down.
-
-Cleaning up resources should be done in `GraphStageLogic.postStop` and not in the `InHandler` and `OutHandler`
-callbacks. The reason for this is that when the stage itself completes or is failed there is no signal from the upstreams
-or the downstreams. Even for stages that do not complete or fail in this manner, this can happen when the
-`Materializer` is shutdown or the `ActorSystem` is terminated while a stream is still running, what is called an
-"abrupt termination".
+../../scala/stream/stream-customize.md
```
Comment (raboof) [inline akka-docs/src/main/paradox/java/stream/stream-customize.md]: Seems something went wrong here - the file was replaced by a file containing the contents "../../scala/stream/stream-customize.md", but it should have been replaced with a symlink. The difference is a bit hard to see in the github UI but should be clear when you build the docs locally. Could you make this a symlink?

**Review by raboof — CHANGES_REQUESTED** [review]

**Comment by atiqsayyed** [comment]
Sorry for the silly thing, updated the PR now :D

File: akka-docs/src/main/paradox/scala/stream/stream-customize.md [hunk 2]
```diff
@@ -233,7 +267,7 @@ in circulation in a potential chain of stages, just like our conceptual "railroa
 Completion handling usually (but not exclusively) comes into the picture when processing stages need to emit
 a few more elements after their upstream source has been completed. We have seen an example of this in our
 first `Duplicator` implementation where the last element needs to be doubled even after the upstream neighbor
-stage has been completed. This can be done by overriding the `onUpstreamFinish` method in `InHandler`.
+stage has been completed. This can be done by overriding the `onUpstreamFinish` method in @scala[`InHandler`] @java[`AbstractInHandler`].
```
Comment (raboof) [inline akka-docs/src/main/paradox/scala/stream/stream-customize.md]: There's a spurious space here for the `scala` version

**Review by raboof — APPROVED** [review]

**Comment by raboof** [comment]
Thanks!

**Pull request merged**